skaffold-operator.yaml

```diff
@@ -86,15 +136,24 @@ build:
         - pkg/extensions
         - pkg/features
         - pkg/gardenlet/apis/config
+        - pkg/gardenlet/apis/config/helper
         - pkg/gardenlet/apis/config/v1alpha1
         - pkg/healthz
         - pkg/logger
+        - pkg/operation
+        - pkg/operation/botanist
+        - pkg/operation/botanist/matchers
+        - pkg/operation/care
         - pkg/operation/common
+        - pkg/operation/garden
+        - pkg/operation/seed
+        - pkg/operation/shoot
```

Comment:
Do you know why we now need all these operation packages (except `care`, of course)? Is there a possibility to get rid of their transitive dependencies?

Reply:
`pkg/operation/care` is the source of most of these dependencies. It is using data types, functions etc. from all over the place for convenience. We could probably move them from here to the respective controllers, but this is some refactoring effort.